packages/tracing/test/browser/request.test.ts

```diff
@@ -383,3 +390,52 @@ describe('callbacks', () => {
     });
   });
 });
+
+// TODO (v8): Adapt these tests once we remove `tracingOrigins`
+describe('[pre-v8]: shouldAttachHeaders', () => {
+  describe('prefer `tracePropagationTargets` over `tracingOrigins`', () => {
+    it('should return `true` if the url matches the new tracePropagationTargets', () => {
+      const shouldAttachHeaders = makeShouldAttachHeaders(['example.com'], undefined);
+      expect(shouldAttachHeaders('http://example.com')).toBe(true);
+    });
+    it('should return `false` if tracePropagationTargets array is empty', () => {
+      const shouldAttachHeaders = makeShouldAttachHeaders([], ['localhost']);
+      expect(shouldAttachHeaders('http://localhost:3000/test')).toBe(false);
+    });
+    it("should return `false` if tracePropagationTargets array doesn't match", () => {
+      const shouldAttachHeaders = makeShouldAttachHeaders(['example.com'], ['localhost']);
+      expect(shouldAttachHeaders('http://localhost:3000/test')).toBe(false);
+    });
+  });
+
+  describe('tracingOrigins backwards compatibility (tracePropagationTargets not defined)', () => {
+    it('should return `true` if the url matches tracingOrigns', () => {
+      const shouldAttachHeaders = makeShouldAttachHeaders(undefined, ['example.com']);
+      expect(shouldAttachHeaders('http://example.com')).toBe(true);
+    });
+    it('should return `false` if tracePropagationTargets array is empty', () => {
+      const shouldAttachHeaders = makeShouldAttachHeaders(undefined, []);
+      expect(shouldAttachHeaders('http://localhost:3000/test')).toBe(false);
+    });
+    it("should return `false` if tracePropagationTargets array doesn't match", () => {
+      const shouldAttachHeaders = makeShouldAttachHeaders(undefined, ['example.com']);
+      expect(shouldAttachHeaders('http://localhost:3000/test')).toBe(false);
+    });
+  });
+
+  describe('should fall back to defaults if no options are specified', () => {
+    it.each([
+      '/api/test',
+      'http://localhost:3000/test',
+      'http://somewhere.com/test/localhost/123',
+      'http://somewhere.com/test?url=localhost:3000&test=123',
+    ])('return `true` for urls matching defaults (%s)', url => {
+      const shouldAttachHeaders = makeShouldAttachHeaders(undefined, undefined);
+      expect(shouldAttachHeaders(url)).toBe(true);
+    });
+    it.each(['notmydoman/api/test', 'example.com'])('return `false` for urls not matching defaults (%s)', url => {
```

Comment:
VL: Do you mind throwing a blank line in between the tests here? Because the test cases are also indented, it makes the break between tests harder to spot at a glance. (I'd actually do it for all the tests, but the other ones are easier to visually differentiate even all smushed together.)

Reply:
[3dd0cce](https://github.com/getsentry/sentry-javascript/pull/6238/commits/3dd0cce62a1df8b385b7613d2497c868102b60c8)